docs/getting-started/getting-started-with-galaxy.rst

```diff
@@ -0,0 +1,81 @@
+
+Getting Started with Galaxy
+===========================
+
+Tutorial Goals
+--------------
+
+-  Learn about `Galaxy <https://training.galaxyproject.org/>`__
+-  Create and run a basic Galaxy workflow
+-  Export the workflow to a file
+-  Setup a GitHub account and repository
+-  Push the workflow to GitHub
+-  Make a GitHub release
+
+Create a basic Galaxy workflow
+------------------------------
+
+Create and run your workflow in Galaxy. Here  is a tutorial for `Creating, Editing, Importing Galaxy Workflows <https://training.galaxyproject.org/training-material/topics/galaxy-interface/tutorials/workflow-editor/tutorial.html>`__
+
+Export the workflow to a file
```

Comment:
Suggest here or in one of the bullet items, explicitly stating "In Galaxy", e.g., "In Galaxy, export the workflow to a file". I was admittedly skimming, but my first thought when I read "Click on the Galaxy UI Workflow link at the top of the page", is where is that in the Dockstore UI?

Reply:
added  'In Galaxy:'